pvsite_datamodel/read/site.py

```diff
@@ -120,6 +123,13 @@ def get_sites_by_country(session: Session, country: str) -> List[SiteSQL]:
     # filter by country
     query = query.filter(SiteSQL.country == country)
 
+    # filter by client string
+    # This could cause an issue if client_a's name is in the site of clients_b.
+    # We could make a new Client table and join it with sites
+    # https://github.com/openclimatefix/pv-site-datamodel/issues/148
```

Comment:
Nice one for tracking potential tech debt!

Reply:
thank you